dlib/media/ffmpeg_abstract.h

```diff
@@ -1013,6 +1013,22 @@ namespace dlib
             !*/
         };
 
+// ---------------------------------------------------------------------------------------------------
+
+        template <typename image_type>
+        std::enable_if_t<is_image_type<image_type>::value, void>
+        load_frame(
```

Comment:
This is already landed on master.  This PR needs to be merged with master I guess?  There are some things about how PRs work on github that are wacky.

Reply:
Oops... Sorry about that, fixed!